pkg/staticnetworkconfig/generator_test.go

```diff
@@ -17,6 +18,39 @@ func TestStaticNetworkConfig(t *testing.T) {
 	RunSpecs(t, "StaticNetworkConfig Suite")
 }
 
+var _ = Describe("Generate Config", func() {
```

Comment:
Nit, but can you make this the name of the method you're testing? i.e. `Describe("generateConfiguration"...`

Reply:
done